packages/web/components/Technology/Details/Tabs/Review/index.js

```diff
@@ -55,7 +62,18 @@ const Review = () => {
 
 	return (
 		<Layout.Cell>
-			<Section title="Relatos" hideWhenIsEmpty={false}>
+			<Section
+				title="Relatos"
+				hideWhenIsEmpty={false}
+				render={() =>
+					rating && (
+						<RatingText aria-label="Média de Avaliações">
+							Avaliação dessa tecnologia: <strong>{rating}</strong>{' '}
+							<Rating value={rating} size={3} readonly />
+						</RatingText>
```

Comment:
Just an obs: aria-label is only needed when the element has no text, like an icon button